python/tritonserver/_c/tritonserver_pybind.cc

```diff
@@ -1282,6 +1282,12 @@ class PyServerOptions : public PyWrapper<struct TRITONSERVER_ServerOptions> {
         triton_object_, thread_count));
   }
 
+  void SetModelLoadRetryCount(unsigned int retry_count)
```

Comment:
Unrelated, but do you think we could have some kind of test that asserts the python bindings fully cover the C API?

I'm thinking long term, folks may not always remember to add the binding equivalent for any API changes, and it might be good to automate that check.